MONEY/NetworkSpendBuySub.md

```diff
@@ -0,0 +1,20 @@
+---
+ns: MONEY
+aliases: ["0x90CD7C6871FBF1B4", "_NETWORK_SPENT_BUY_SUB"]
```

Comment:
Hi, why is there a `_NETWORK_SPENT_BUY_SUB` alias here? This is not present in the previous native name 🤔

Reply:
It's the "old name" of this native found in other website such as nativedb, i'm going to remove it